utils/install_latest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+if [ "$(id -nu)" != "grafana" ]; then
+  # see https://github.com/grafana/grafana/issues/25367
+  echo "ERROR: This script has to be run as grafana user! you may use 'sudo -u grafana bash install_latest.sh'"
```

Comment:
This does not always work. In my case, I got a permission denied. It's most likely, because the grafana user has very limited permissions.

Reply:
The mentioned issue indicates, that there is only a problem, if the plugin is installed before the server has been started for the first time. I could not reproduce the error (Grafana 9.3.6). But if this is a real problem, we should take a look, if the server has been started at least once (or is running) and deny the installation as root otherwise?

Reply:
I just tested it with ubuntu 22.04: The installation does not start grafana, so i executed the script without this check as root, and I could still use the plugin.  But the plugin is still owned by root:
```
root@ubuntu2204:~# ls -lah /var/lib/grafana/plugins/
total 12K
drwxr-xr-x 3 root    root    4.0K Mar 15 12:11 .
drwxr-xr-x 7 grafana grafana 4.0K Mar 15 12:15 ..
drwxr-xr-x 3 root    root    4.0K Mar 15 12:11 tribe-29-checkmk-datasource
```
this might lead to problems later on?

Reply:
as we just discussed: this leads to the situation, that you can no longer install plugins via the web ui, as the plugin dir is owned by root, and the grafana user can not write to this folder.

Reply:
the script will now change to the grafana user if executed as root.